ark/utils/example_dataset.py

```diff
@@ -154,7 +154,7 @@ def get_example_dataset(dataset: str, save_dir: Union[str, pathlib.Path],
 
     example_dataset = ExampleDataset(dataset=dataset, overwrite_existing=overwrite_existing,
                                      cache_dir=None,
-                                     revision="main")
+                                     revision="22c044ac9a62200849e0f66a83aa2dfd30a2c56e")
```

Comment:
This will be changed back to `'main'` prior to merging into the `main` branch.

Reply:
Looks good